src/main/java/pl/javastart/task/contracts/MixContract.java

```diff
@@ -0,0 +1,65 @@
+package pl.javastart.task.contracts;
+
+public class MixContract extends Contract {
```

Comment:
Mix ma wiele cech wspólnych z CardContract, powinieneś dziedziczyć właśnie po tej klasie.

W tej klasie logika powinna być taka, że np. przy wysyłaniu smsa:
- sprawdzamy, czy mamy jeszcze dostępne sms w pakiecie,
- jeżeli nie, to sprawdzamy, czy mamy środki na koncie
- jeżeli nie, to nie można wysłać sms.

Przy rozmowach jest trochę bardziej skomplikowana sytuacja, bo można np. użyć 2 minuty z pakietu, później 2 minuty z środków na koncie, a dodatkowo może nam zabraknąć środków do wykonania pełnej rozmowy